<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab_page">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>691</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Photométrie</string>
     </attribute>
     <widget class="QLabel" name="label_photometrie_img">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_photo_img">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Opérations :</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Négatif</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Luminosité</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Exploitation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Egalisation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normalisation</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="combo_photo_histo">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>390</y>
        <width>200</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Histogramme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Histogramme cumulé</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_histogramme">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>46</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Aperçu :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Valid_photometrie">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>390</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QSlider" name="Slider_luminosite">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>510</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="sliderPosition">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="annul_photometrie">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>420</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="histo_blue">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>430</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Bleu</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="histo_red">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Rouge</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="histo_green">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>430</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Vert</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="histo_gray">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>430</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Gris</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Go !</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Arithmétique</string>
     </attribute>
     <widget class="QLabel" name="label_arithmetique_img">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_arithmetique_res">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_arith_img_1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Choisissez votre seconde image</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_arith">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>430</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Opérations :</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Addition</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Soustraction</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multiplication</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Division</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Maximum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Minimum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ET</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OU</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Valid_arith">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>430</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_largeur_arith">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Largeur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_hauteur_arith">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hauteur :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Annul_arith">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Géométrique</string>
     </attribute>
     <widget class="QLabel" name="label_apercu_geo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aperçu :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_geo_img">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Valid_geo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_geo">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>390</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Opérations :</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Extraction</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Retournement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rotation</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Annul_geo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>420</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_retournement_vert">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>500</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_retournement_hor">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>520</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_retournement_verthor">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>540</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Les deux</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_rotation_90">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>420</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>90°</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_rotation_180">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>180°</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_rotation_270">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>460</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>270°</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_rotation_x">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Angle (°):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rotation_x">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>480</y>
        <width>31</width>
        <height>18</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>999</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Conversion</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Options</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hauteur_img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>hauteur : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_largeur_img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>largeur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Votre Image :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_photo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choisissez votre image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_save">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer votre image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_format_img">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Format :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nbbit_img">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de bits :</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tab_page</tabstop>
  <tabstop>combo_photo_img</tabstop>
  <tabstop>Valid_photometrie</tabstop>
  <tabstop>combo_photo_histo</tabstop>
  <tabstop>Slider_luminosite</tabstop>
  <tabstop>annul_photometrie</tabstop>
  <tabstop>histo_blue</tabstop>
  <tabstop>histo_red</tabstop>
  <tabstop>histo_green</tabstop>
  <tabstop>histo_gray</tabstop>
  <tabstop>btn_photo</tabstop>
  <tabstop>Button_save</tabstop>
  <tabstop>Annul_arith</tabstop>
  <tabstop>btn_arith_img_1</tabstop>
  <tabstop>Valid_geo</tabstop>
  <tabstop>combo_geo</tabstop>
  <tabstop>Annul_geo</tabstop>
  <tabstop>radio_retournement_vert</tabstop>
  <tabstop>radio_retournement_hor</tabstop>
  <tabstop>radio_retournement_verthor</tabstop>
  <tabstop>radio_rotation_90</tabstop>
  <tabstop>radio_rotation_180</tabstop>
  <tabstop>radio_rotation_270</tabstop>
  <tabstop>radio_rotation_x</tabstop>
  <tabstop>rotation_x</tabstop>
  <tabstop>combo_arith</tabstop>
  <tabstop>Valid_arith</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
